packages/react-client/src/ReactFlightClient.js

```diff
@@ -24,7 +24,13 @@ import {
   parseModel,
 } from './ReactFlightClientHostConfig';
 
-import {REACT_LAZY_TYPE, REACT_ELEMENT_TYPE} from 'shared/ReactSymbols';
+import {getOrCreateServerContext} from 'react/src/ReactServerContext';
```

Comment:
This is a cross-package source import. Can't do it like this because the `react` package gets compiled. So this file is in the compiled `react` bundle, and you can't import it anymore. Usually there are two options:

1. If the module is not stateful, you can put it into `shared` and import from both packages to literally duplicate its code in both packages. We've been trying to avoid adding more of these though. Also, I think your module _is_ stateful due to `globalRegistry`.
2. Alternatively, you'd have to expose it via SECRET_INTERNALS. This is what we do for other stateful things like current dispatcher:

https://github.com/facebook/react/blob/532dfa28d4244dd278425778cbf90b05cd2a2064/packages/react/src/ReactSharedInternals.js